test/Validation.test.js

```diff
@@ -187,6 +188,6 @@ describe("Validation", function() {
 				return;
 			}
 			throw new Error("Validation didn't fail");
-		})
+		});
```

Comment:
Add two testcases which tests the error message when a non-/absolute is provided.